### ISSUE TITLE
во время реквеста добавился новый ордер в ордербук почемуто анимация показала что запросили его (видео)

### ISSUE THREAD

**Issue by noxonsu** [swaponline/swap.react#1074]
https://youtu.be/lvjayJgOzxI 
0:15 человек нажимает на начало свапа
0:17 появляется новый свап и почему то реквест перекидывается на него
!! и при этом реквестинг показывает на нижний https://airdb.wpmix.net/chrome_2018-12-03_14-14-41202cbbac-c96f-41ed-956f-769815987bc0.png 

0:20 появляется третий ордер и реквест перекидывается на него
0:37 видно что начали свап с последний появившимся

**Comment by noxonsu** [comment]
стелс баг, не могу повторить на своей стороне

**Comment by caffeinum** [comment]
подозреваю, что связано с партиалс

**Comment by noxonsu** [comment]
предлагаю пока сделать так
1. при отправке реквеста записывать адрес боба, сумму отдать-получить в переменную
2. при переходе в свап дополнительно проверить это

**Comment by shendel** [comment]
Так оно и есть... я писал про это, но меня не услышали
https://github.com/swaponline/swap.react/commit/6c8ac6837288b49480765734c583bd88bdc48957#diff-2933f5de07c739e4ff975355fc3bdaab

**Comment by caffeinum** [comment]
@shendel я так понимаю, это даже не партиал ордер был, на скриншоте

**Comment by shendel** [comment]
@caffeinum не факт. По коду в ордер-буке показывает все - (партиал/не партиал/ответка). В определенный момент (по истечении 10 секунд), когда создается ответка, ее видят другие пользователи (Сара, Сьюзи, Джен и Бен) и они могут отправить запрос на свап - в это время у Алисы (которая продает частями) показывает два уведомления (с таймером) на свап - от Боба и от того "кто первый встал"
А дальше уже по ситуации, на какой запрос Алиса даст согласие....

**Comment by shendel** [comment]
@caffeinum пройдет ли свап? не факт, что будет как в ишью (мне кажется что будет с тем, кто первый "встал" - по коду ничего не мешает), но проблема на лицо

**Comment by shendel** [comment]
@caffeinum да, ты прав... тут другая проблема (тоже давняя) - во время активного свапа можно принять запрос на другой свап.... и если "прозевать", то может что-то пойти не так (перенаправит на другой свап, а тот не начнется из-за не завершенного первого свапа (isSwapExists или как там эта функция в контракте))

**Comment by nikdementev** [comment]
на скринах, это обычные ордера к partials никакого отношения не имеют)
